chromeBinary.dot

```diff
@@ -0,0 +1,9 @@
+digraph versiontest {
+
+  input [data="/usr/local/google/home/soonm/tree-builder-builder/example.ejs"]
+  chromeBinary [version="323860", chromium="/usr/local/google/home/soonm/chromium/src", platform="linux"]
+  telemetryPerf [chromium="/usr/local/google/home/soonm/chromium/src", browser="system", perfBrowser="system"]
+
+  input -> ejsFabricator -> chromeBinary -> telemetryPerf -> traceTree -> tracePrettyPrint -> consoleOutput;
```

Comment:
This needs to import trace-phases.js to use traceTree, see trace.dot for an example.